src/storage/v3/shard_rsm.cpp

```diff
@@ -922,7 +820,7 @@ msgs::ReadResponses ShardRsm::HandleRead(msgs::ScanVerticesRequest &&req) {
     }
   }
 
-  msgs::ScanVerticesResponse resp{};
+  memgraph::msgs::ScanVerticesResponse resp{};
```

Comment:
I think we are in the `memgraph` namespace, so this is not necessary.

Reply:
I've removed it from the whole file.